pkg/backup/client.go

```diff
@@ -748,8 +748,23 @@ func SendBackup(
 	return nil
 }
 
+// ChecksumMatches tests whether the "local" checksum matches the checksum from TiKV.
+func (bc *Client) ChecksumMatches(local []uint64) bool {
+	if len(local) != len(bc.backupMeta.Schemas) {
+		return false
+	}
+
+	for i, schema := range bc.backupMeta.Schemas {
+		if local[i] != schema.Crc64Xor {
```

Comment:
The checksum should compare all of crc64xor, total bytes and total kvs.

Reply:
Yep, I'm dealing with that now.